lib/infrastructure_collector.rb

```diff
@@ -273,4 +268,20 @@ def hosts_filter_spec(starting_folder = VSphere::root_folder, properties = [])
     )
   end
 
+  def convert_speed_mbits_to_speed_bits_per_second(nics)
+    result = {}
+    nics.each do |key,value|
+      new_value = []
+      value.map do |val|
+        if val[:speed_mbits]
+          val.merge!({speed_bits_per_second: (val[:speed_mbits] * 1000000)})
```

Comment:
I know that this value may not change (100000) but, can we try to use a Constant instead of the number?